src/box_utils.cpp

```diff
@@ -59,6 +60,19 @@ std::string boxToString(const Box& box)
   return box_stringstream.str();
 }
 
+std::string boxArrayToString(const std::vector<Box>& box_array)
+{
+  std::stringstream box_array_stringstream;
+  box_array_stringstream << "[";
+  for (Box box : box_array)
+  {
+    box_array_stringstream << boxToString(box);
```

Comment:
We don't need "," between boxes?

Reply:
Although VVVF parser does not concern whether commas exist there, you are right, it is better to add commas between box definitions.